src/lib.rs

```diff
@@ -33,9 +33,11 @@ pub mod efuse;
 #[cfg(feature = "external_ram")]
 pub mod external_ram;
 pub mod gpio;
+#[cfg(feature = "rt")]
 pub mod interrupt;
 pub mod prelude;
 pub mod serial;
+#[cfg(feature = "rt")]
```

Comment:
Why does the timer module need to be excluded. It doesn't use any RT features as far as I can see.

Reply:
It uses the `interrupt::free` function.

Reply:
`interrupt::free` has been moved to `xtensa-lx6`, so doesn't depend on `xtensa-lx6-rt`.

Reply:
I moved the `cfg` attributes into the `interrupt` module now.